integration_test/third_party_apps_data/applications/flink/debian_ubuntu/install

```diff
@@ -1,7 +1,10 @@
 set -e
 
 sudo apt-get update
-sudo apt-get install -y curl default-jre
+
+# All other tested distributions have default-jre point to openjdk-11-jre, but default-jre pulls in openjdk-17-jre
+# for Ubuntu 23.04, and Java 17 support is in progress for Flink (see https://issues.apache.org/jira/browse/FLINK-15736)
+sudo apt-get install -y openjdk-11-jre wget
```

Comment:
Interesting, you no longer install `curl` explicitly — I'm guessing that's because it's already present on the system. Why do you need `wget`, though? No changes requested, just curious…

Reply:
Just a copy paste issue from hbase, oops. Will change it back to explicit curl.